src/modules/Dashboard/components/FlowSummaryCard/FlowSummaryThumbnail/FlowSummaryThumbnail.styles.ts

```diff
@@ -0,0 +1,43 @@
+import { Box, styled } from '@mui/material';
+
+import { theme, variables } from 'shared/styles';
+
+export const StyledRoot = styled(Box)({
+  background: variables.palette.primary_container,
+  borderRadius: 8,
+  display: 'flex',
+  flexShrink: 0,
+  flexWrap: 'wrap',
+  height: theme.spacing(8),
+  overflow: 'hidden',
+  placeContent: 'flex-start',
+  placeItems: 'flex-start',
+  position: 'relative',
+  width: theme.spacing(8),
+});
+
+export const StyledActivityThumbnail = styled('img')({
+  background: variables.palette.surface1,
+  width: '50%',
+  height: '50%',
+  position: 'relative',
+  objectFit: 'cover',
+  border: 'none',
+});
+
+export const StyledItemCount = styled('div')({
```

Comment:
Nit: It's common elsewhere to use `StyledFlexAllCenter` for center-aligned content, so you should be able to get rid of the `display`, `placeContent`, `placeItems` props (if you want).

Side note: I've noticed you you habitually use `place-content` over `justify-content`, and `place-items` over `align-items`. I'm not used to using them. I get that they're shorthands, but I suppose it doesn't occur to me to affect both axes by default – but maybe it's considered best practice these days and I'm just out of the loop! 😅 I'll switch to that if you think it's a good habit?

Reply:
I'll be honest and say that I've started using it by default not out of a need for specify both axes/block and inline elements simultaneously, and more because I appreciate the synonymous nomenclature (ie, I typically sort lists of properties and it allows for these similar styles to get sorted together for easy reference).

If there were a real need to explicitly declare justify _and_ align for content _and_ items (which I've never really run into since I sorta fell into using `placeX` by default), I'd be inclined to use the explicit properties for the things I want to set.